features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml

```diff
@@ -1,20 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="com.google.cloud.tools.eclipse.suite.e45.feature"
-      label="Google Cloud Platform for Eclipse 4.5"
+      label="Cloud Tools for Eclipse 4.5"
       version="0.1.0.qualifier"
       provider-name="Google, Inc.">
 
-   <description>
-      Support development of Google Cloud Platform-based Projects in Eclipse 4.5
-   </description>
+  <description>
+    Support development of Google Cloud Platform-based Projects in Eclipse 4.5
+  </description>
 
-   <copyright>
-      Copyright 2016
-   </copyright>
+  <copyright>
+    Copyright 2016
+  </copyright>
 
-   <license>
-      Todo
+  <license>
+    Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Check out this commit: https://github.com/GoogleCloudPlatform/google-plugin-for-eclipse/pull/118/commits/cdc02aad2c3d2e7ed95c55a1f990a7e12222ed38 I think we should use the license feature provided by Eclipse instead of adding the license contents explicitly here.